lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -44,7 +46,7 @@
                 },
 
                 render: function () {
-                    this.$el.html(_.template(teamsTemplate));
+                    HtmlUtils.setHtml(this.$el, HtmlUtils.HTML(teamsTemplate));
```

Comment:
I see. You are right that in this case the template is just an HTML string. The problem with this is that if the template is changed to dynamically inject values, e.g. if it uses gettext to put in localized strings, then it will break. There's no way from reading this code to know that it is safe to render the template directly, which means the reviewer of the code has to open the template file and verify that it is indeed safe. It is better IMO to always render templates even if they don't have any dynamic values at the moment.